README

```diff
@@ -1,13 +0,0 @@
-Copyright (C) 2011-2015 lishid.  All rights reserved.
```

Comment:
No worries :D I added it back to the end of the README, so all is well. Thanks again for this pull!